### ISSUE TITLE
Disable CLI xdebug in DrupalVM

### ISSUE THREAD

**Issue by danepowell** [acquia/blt#2034]
For background, see https://github.com/acquia/blt/pull/1983

For performance reasons, we should completely unload xdebug for PHP CLI in DrupalVM. I'm not sure this is actionable right now, but I want to keep this issue open for tracking.

This will be possible once this is merged upstream: https://github.com/geerlingguy/ansible-role-php-xdebug/pull/54

After that's merged, we may have to add this to our default box config.yml (depending on whether DrupalVM decides to make it a default or not):
`php_xdebug_cli_disable: yes`

Theoretically, BLT could distribute a patched version of DrupalVM to take advantage of this change now, but it would be a pain given how BLT manages the DrupalVM dependency, so probably not worth it.

**Comment by geerlingguy** [comment]
I'm hoping to get that new variable working for Red Hat-based OSes as well as Debian-based, so it could be a little longer before the option is merged into a tagged Drupal VM release (next minor version cutoff is 'soon').
